### ISSUE TITLE
Tugboat always Migrating instead of Keeping

### ISSUE THREAD

**Issue by AndreLobato** [metocean/tugboat#25]
When I set env var or expose ports on the tug file, then doing a tug up, it will always Migrate, even if I didn' t changed the tug file. Looks like tug is getting the setting from the file correctly but when comparing is always finding diffs.
e.g.
```
$ tug diff SchedulerAll.yml

 Diff of SchedulerAll...

  postgres                         environment different (PGDATA=/db, HOST_HOSTNAME=katipo, PATH=/usr/lib/postgresql/9.3/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin, LANG=en_US.utf8, PG_MAJOR=9.3, PG_VERSION=9.3.6-1.pgdg70+1 -> PGDATA=/db, HOST_HOSTNAME=katipo, PATH=/usr/lib/postgresql/9.3/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin, LANG=en_US.utf8, PG_MAJOR=9.3, PG_VERSION=9.3.6-1.pgdg70+1, PGDATA=/var/lib/postgresql/data)
  postgres                         Migrating SchedulerAll_celerybeat_1/postgres
  flower                           expose different (2 items -> 1 items)
  flower                           Migrating SchedulerAll_flower_1
...
```
My config file says:

```
postgres:
    image: metocean/postgres
    ports:
        - "5432:5432"
    volumes:
        - /tmp/db:/db
        - ../consul/scheduler/postgres.json:/consul/postgres.json
    environment:
        PGDATA: /db

flower:
    image: metocean/ops-base:testing
    user: metocean
    ports:
        - "80:5555"
    links:
        - rabbitmq
        - postgres
    volumes:
        - ../consul/scheduler/flower.json:/consul/flower.json    
 
```

So what seems to be misleading Tug are the EXPOSE and ENV statements on the Dockerfiles.

**Comment by ajnisbet** [comment]
Fixed by e6782524acff952f94e2fd1ee6524259aadcb678

**Comment by ajnisbet** [comment]
Some notes:
* this might happen again with some of the other docker/tugboat dual settings
* it doesn't look like environment works as a global config options
